### ISSUE TITLE
How to save the input address?

### ISSUE THREAD

**Issue by paulgaumer** [Grsmto/sanity-plugin-mapbox-input#4]
Hi @Grsmto ,

Thanks a lot for this cool plugin! I am building a dashboard where the user inputs an address and both the address and the map with a marker display on screen. I noticed the plugin only saves the lat & lng, which means my user currently needs the type the address once in another field to store it and a second time in this plugin's field to display the marker on the map.

Is there an easy way to also store the address typed in the mapbox input for later use?

I'd tweak it myself but I've never worked with plugins and I'm not too sure where to start...

**Comment by Grsmto** [comment]
Hi Paul,
This plugin is specifically to replace the Sanity `geopoint` type which is meant to store a lat/lng, not a full address etc.
What I did when I used this plugin myself is that I had an extra field for the address, exactly like you described.

To do what you want you'll need to create a custom plugin, it's not too complicated if you check Sanity tutorials.
You'll have somehow to change the code [there](https://github.com/Grsmto/sanity-plugin-mapbox-input/blob/master/src/GeopointSelect.js#L77) so it saves the address of the geocoder, right now it saves only the lat/lng.
